Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -20,8 +20,8 @@ class Client(BaseClient):
     """
 
     def __init__(self, server_url, use_ssl, proxy, reliability, feed_tags, tlp_color, content_max_size=45,
-                 read_timeout=20, enrichment_excluded=False):
-        super().__init__(base_url=server_url, proxy=proxy, verify=use_ssl, headers={'User-Agent': USER_AGENT})
+                 read_timeout=20, enrichment_excluded=False, headers={}):
```

Comment:
Please don't init with a `dict` value - better to set it to `None`, and set its value in the code via an `if` condition.

Reply:
done